### PR TITLE
Larger Team Update

### PR DESCRIPTION
**Pull request by mmmries**
The Spiff engineering team has grown a lot since the last changes to this document. This PR represents a lot of feedback from a lot of different people. It's my best attempt to make this document more relevant to life on a larger and more established product and engineering team.

### CODE REVIEW

File: 02.SettingYourCourse.md [hunk 1]
```diff
@@ -4,22 +4,30 @@ When you drive to work, you don't spend 30 mins perfectly aiming your car at pre
...
-Can you help sales people better understand their commission plan?
+## Picking A Destination
+
+A good place to start is to check your pod's OKRs and Roadmap items.
+These objectives have been discussed between engineering and product and take into account company goals as well as the specific users that are impacted by the work of your pod.
+If you believe that the pod objectives are the right things to focus on, then you might pick a few of those as your personal goals.
+
+> If you suspect your pod objectives are focused on the wrong things, you should prepare an argument for how to improve them and discuss with your manager and/or product manager.
+
+There are also times where engineers identify initiatives that are orthogonal to the goals of their pod.
```
Comment (skilletspecial) [inline 02.SettingYourCourse.md]: I recommend using something easier to understand than "orthogonal" -- especially with many devs who speak English as a second language.  Maybe "independent" or "unrelated"?  And if those alternatives don't convey the same meaning I still think it's worth finding something more accessible.

File: 02.SettingYourCourse.md [hunk 2]
```diff
@@ -4,22 +4,30 @@ When you drive to work, you don't spend 30 mins perfectly aiming your car at pre
...
+Should we establish better testing patterns for behavior that depends on the output of a statement calculation?
+Perhaps life would be better on your pod if started a weekly tradition of posting Nicholas Cage memes?
+
+## Measure Your Progress
+
+Your goals should be focused on the long-term success of the people using the software you build.
```
Comment (skilletspecial) [inline 02.SettingYourCourse.md:21]: This is addition is great!

File: 05.CheckingIn.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 As you get used to a new team, a new product, and maybe some new tools, it is useful to get your bearings from time to time.
 Check-ins are a way for Spiff to know what is important to you and for you to know what is important to Spiff.
-Check-ins are a one-on-one meeting between you and the CTO that happen every 3 months.
-If you forget, the CTO will remind you. If the CTO forgets, you should remind them.
+Check-ins are a one-on-one meeting between you and your manager that happen at least once per quarter.
```
Comment (skilletspecial) [inline 05.CheckingIn.md]: A single 1:1 with your manager seems very infrequent.  Once a quarter may make sense for a skip-level 1:1 but I would hope everyone is meeting with their manager more frequently.

**Review by skilletspecial — COMMENTED** [review]
I really like these changes!  There are typos and unfinished thoughts which I didn't call out (I would rather focus on the content at this point) but I appreciate the direction the document is moving in.

File: 05.CheckingIn.md (hunk 3 above)
Comment (mmmries) [reply 05.CheckingIn.md]: I agree, let's document this as once-per-month? That's the longest I can imagine doing it if there was someone who was already established on the team and was wanting to minimize meetings

File: 02.SettingYourCourse.md (hunk 1 above)
Comment (mmmries) [reply 02.SettingYourCourse.md]: Change it to "identify initiatives that are separate from the goals of their pod" 👍

**Pull request merged**